RxCocoa/iOS/UITableView+Rx.swift

```diff
@@ -168,15 +168,6 @@ extension Reactive where Base: UITableView {
 extension UITableView {
  
     /**
-    Factory method that enables subclasses to implement their own `delegate`.
-    
-    - returns: Instance of delegate proxy that wraps `delegate`.
-    */
-    public override func createRxDelegateProxy() -> RxScrollViewDelegateProxy {
```

Comment:
Hi @volodg ,

I've missed these changes. I looks like these cause real issues.

Could you please explain why you've removed these `createRxDelegateProxy` method overrides?